src/common/config.cc

```diff
@@ -713,6 +713,9 @@ int md_config_t::parse_argv(ConfigValues& values,
     else if (ceph_argparse_witharg(args, i, &val, "--client_mountpoint", "-r", (char*)NULL)) {
       set_val_or_die(values, tracker, "client_mountpoint", val.c_str());
     }
+    else if (ceph_argparse_witharg(args, i, &val, "--luarocks_location", "-r", (char*)NULL)) {
```

Comment:
This argument needs documented. I would think this would go in the rgw code too as it's not really common to all daemons.

Reply:
ok. will move it elsewhere.

Reply:
removed from here